runbooks/workflow/creating-maintaining-littlepay-data-syncs.md

```diff
@@ -0,0 +1,51 @@
+# Creating and Maintaining Littlepay Data Syncs
+
+> Some of this is taken from the provided Littlepay documentation, with Cal-ITP specific content added.
```

Comment:
Can we link to the provided Littlepay documentation? If it's behind authentication (either in Cal-ITP GDrive or the Littlepay site login) we can just make a note of that.

Reply:
(I am wary of reproducing instructions that exist elsewhere in case they get out of sync)

Reply:
Resolved in [0049f24](https://github.com/cal-itp/data-infra/pull/3019/commits/0049f24032f29d2a243d2db04fe472dcd4067ae7), though the existing instructions here were so entangled with/customized on top of the Littlepay instructions that there's still some copy/pasted material in this document.